src/Compiler/SyntaxTree/XmlDoc.fs

```diff
@@ -320,14 +320,14 @@ type XmlDocumentationInfo private (tryGetXmlDocument: unit -> XmlDocument option
         )
 
     let tryGetSummaryNode (xmlDocSig: string) =
-        if xmlDocSig.Contains "'" && xmlDocSig.Contains "\"" then
+        if xmlDocSig.Contains ''' && xmlDocSig.Contains '\"' then
```

Comment:
The first will probably need escaping?

Reply:
Yeah, but why it did compiled?

Reply:
Interesting, perhaps there's some special case in parser for it.

https://sharplab.io/#v2:DYLgZgzgNAJiDUAfYBTALgAgB4YLwYHIig==

I didn't know that